types/markdown-it-external-links/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for markdown-it-link-attributes 0.0
+// Project: https://github.com/rotorz/markdown-it-external-links
+// Definitions by: Daniil Pronin <https://github.com/grawl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { PluginWithOptions } from 'markdown-it';
+import { RenderRule } from 'markdown-it/lib/renderer';
+
+declare namespace markdownItExternalLinks {
+    interface Config {
+        externalClassName?: string | null;
+        internalClassName?: string | null;
+        internalDomains?: string[];
+        externalTarget?: string;
+        internalTarget?: string;
+        externalRel?: string;
+        internalRel?: string;
```

Comment:
```suggestion
        internalDomains?: string[] | undefined;
        externalTarget?: string | undefined;
        internalTarget?: string | undefined;
        externalRel?: string | undefined;
        internalRel?: string | undefined;
```
this is optional. I've checked the code and it uses '||'  expressions to get values, so this will work with undefined providing default values:
https://github.com/rotorz/markdown-it-external-links/blob/master/lib/index.js
this would allow to support exact optional properties options when using this library in DT ecosystem:
https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types
thx!